### DISCUSSION TITLE
how to disable/fix finder status icons?

### DISCUSSION THREAD

**Discussion by daviehh** [iterate-ch/cyberduck#16696] [Q&A] [answered]
In a recent update, now there is a progress icon in finder 
https://github.com/iterate-ch/cyberduck/issues/16568

however, it can be quite buggy. For example, for uploading to ssh / sftp to a local nas,
the icons often do not refresh properly, which can be almost always reproducible when dragging a folder with many files to cyberduck, leading to many 'in progress' icons after the transfer is done and do not go away until quitting cyberduck:

<img width="629" alt="Screenshot 2024-12-19 at 20 43 35" src="https://github.com/user-attachments/assets/84f84c5e-c42c-47af-8e08-fab8fbf92968" />

**Comment by dkocher** [comment, marked as answer]
You can disable it using the [hidden properties](https://docs.cyberduck.io/tutorials/hidden_properties/)

```
queue.download.icon.update=false
queue.upload.icon.update=false
```

**Reply by daviehh** [reply]
Thanks!
